subsys/git/src/main/java/org/commonjava/indy/subsys/git/GitManager.java

```diff
@@ -261,20 +245,21 @@ public GitManager addAndCommitPaths( final ChangeSummary summary, final Collecti
 
         try
         {
-            final AddCommand add = git.add();
-            final CommitCommand commit = git.commit();
-            for ( final String filepath : paths )
+            synchronized ( git )
```

Comment:
I recommend to user repo as git, which is more meaningful I think. This is not a big deal, depends on you.